integration_tests/test_python_plugin.py

```diff
@@ -84,6 +87,48 @@ def test_build_rewrites_shebangs(self):
         self.assertEqual('#!/usr/bin/env python3', python3_shebang)
         self.assertEqual('#!/usr/bin/env python3', python_shebang)
 
+    def test_pbr_console_scripts(self):
+        """Verify that LP: #1670852 doesn't come back."""
+        # pbr needs git repos to work
+        def pre_func():
+            for python_dir in ['python2', 'python3']:
+                d = os.path.join(self.path, python_dir)
+                check_output(['git', 'init'], cwd=d)
+                check_output(
+                    ['git', 'config', 'user.name', 'Test User'], cwd=d)
+                check_output(
+                    ['git', 'config', 'user.email', '<test.user@example.com'],
+                    cwd=d)
+                check_output(['git', 'add', '.'], cwd=d)
+                check_output(['git', 'commit', '-m', 'initial'], cwd=d)
```

Comment:
I didn't have a split API to get this done. Let's create a bug and get it done in the future. I sort of prefer this mechanism though.